setup.py

```diff
@@ -102,7 +99,7 @@ def get_extras_require():
             # See https://github.com/optuna/optuna/issues/565 for further details.
             'tensorflow<2.0.0',
             'torch',
-            'torchvision'
+            'torchvision>=0.5.0'
```

Comment:
Would it make sense to omit this version constraint, considering the fact that it's not really an Optuna issue? The previous `pillow<7` was mostly a hotfix to work around an issue confined to torchvision.

Reply:
It's not directly an Optuna problem, but users could run in to it when installing Optuna.

With no constraints, they could already have torchvision<0.5.0 and Pillow>=7.0.0, and they'll hit the problem.

So if you want to be sure to avoid that when users install Optuna:

1) torchvision uses `PIL.__version__`:

```
'torchvision>=0.5.0',
```

or

2) torchvision uses `PIL.PILLOW_VERSION`, still in Pillow:

```
'pillow<7',
'torchvision',
```

or

3) torchvision doesn't yet use `PIL.PILLOW_VERSION` or `PIL.__version__`:

```
'torchvision<0.2.1',
```

Reply:
Sorry for the delayed response. Thanks for the alternatives. I just noticed that these requirements are mostly meant for CI (`extras_require`) and won't affect the general users so I think it's okay to keep your suggested changes.